packages/client/src/peer-discovery/peer-discovery.ts

```diff
@@ -133,20 +141,9 @@ export class PeerDiscovery {
 
                 if (port >= 1 && port <= 65535) {
                     const peerData: IPeer = {
```

Comment:
Some more nitpicking :sweat_smile: 
This code to create `peerData` can be also deleted, 
leaving 
`peers.push(peer);` is enough